docs/quickstarts/ratify-with-venafi.md

```diff
@@ -0,0 +1,209 @@
+
+## Deploy Ratify
+
+Now we can deploy Ratify to our cluster:
+```shell
+helm install ratify \
+    ratify/ratify --atomic \
+    --namespace gatekeeper-system \
+    --set featureFlags.RATIFY_EXPERIMENTAL_DYNAMIC_PLUGINS=true \
+    --set featureFlags.RATIFY_CERT_ROTATION=true
+```
+Refer to the Ratify [documentation](https://ratify.dev/docs/1.0/quickstarts/quickstart-manual/) if you need to customize the helm chart installation.
+
+After deploying Ratify, we will download the Venafi CodeSign Protect notation plugin to the Ratify pod using the [Dynamic Plugins feature](../reference/dynamic-plugins.md):
+
+```shell
+cat > venafi-notation-plugin.yaml << EOF
+apiVersion: config.ratify.deislabs.io/v1beta1
+kind: Verifier
+metadata:
+  name: notation-venafi-csp-plugin
+spec:
+  name: notation-venafi-csp
+  artifactTypes: application/vnd.oci.image.manifest.v1+json
+  source:
+    artifact: ghcr.io/venafi/notation-venafi-csp:linux-amd64-latest
+EOF
+
+kubectl apply -f venafi-notation-plugin.yaml
+```
+
+Next we will need to deploy the trusted Root certificate used to issue the signing identity referenced above:
+
+```shell
+cat > venafi_root.yaml << EOF
+apiVersion: config.ratify.deislabs.io/v1beta1
+kind: CertificateStore
+metadata:
+  name: ratify-notation-ztpki-cert
+  namespace: gatekeeper-system
+spec:
+  provider: inline
+  parameters:
+    value: |
+      -----BEGIN CERTIFICATE-----
+      ...
+      -----END CERTIFICATE-----
+EOF
+
+kubectl apply -f venafi_root.yaml
+```
+
+Finally, we will create a verifier that specifies the trust policy to use when verifying signatures. In this guide, we will use a trust policy that only trusts images signed by the Venafi CodeSign Protect signing identity we created earlier:
+
+```shell
+cat > notation-verifier.yaml << EOF
+apiVersion: config.ratify.deislabs.io/v1beta1
+kind: Verifier
+metadata:
+  name: verifier-notation
+spec:
+  name: notation
+  artifactTypes: application/vnd.cncf.notary.signature
+  parameters:
+    verificationCertStores:
+      certs:
+        - ratify-notation-inline-cert
+    trustPolicyDoc:
+      version: "1.0"
+      trustPolicies:
+        - name: default
+          registryScopes:
+            - "*"
+          signatureVerification:
+            level: strict
+          trustStores:
+            - signingAuthority:certs
+          trustedIdentities:
+            - "x509.subject: CN=signer.example.com,O=Acme,L=Cupertino,ST=CA,C=US"
+EOF
+
+kubectl apply -f notation-verifier.yaml
+```
+
+More complex trust policies can be used to customize verification. See [notation documentation](https://github.com/notaryproject/notaryproject/blob/main/specs/trust-store-trust-policy.md#trust-policy) for more information on writing trust policies.
+
+## Deploy Container Image
+
+Now that the signed container image is in the registry and Ratify is installed into the Kubernetes cluster we can deploy our
+container image:
+
+```shell
+kubectl run demosigned --image $REPO_URI:v1
```

Comment:
nit: it might be better to explicitly specify the "default" namespace since the constraint used in the guide applies only to the "default" namespace. Same applies to the unsigned command below.